### ISSUE TITLE
DB4S Crashes on Modify

### ISSUE THREAD

**Issue by Vex-atioN** [sqlitebrowser/sqlitebrowser#1611]
### Details for the issue

I click on a table of my database, then click "Modify", then when I try to modify the data type of a column the whole program gets a Segmentation Fault and crashes. 

#### I'm opening this issue because:

- [x] DB4S is crashing
- [ ] DB4S has a bug
- [ ] DB4S needs a feature
- [ ] DB4S has another problem

#### I'm using DB4S on:

- [ ] Windows: ( _version:_ ___ )
- [x] Linux: ( _distro: Arch Linux)
- [ ] Mac OS: ( _version:_ ___ )
- [ ] Other: ___

#### I'm using DB4S version:

- [ ] 3.11.0-alpha*
- [x] 3.10.1
- [ ] Other: ___

#### I have also:

- [ ] Tried out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [x] Searched for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by chrisjlocke** [comment]
The issue doesn't sound familiar.  However, 3.10.1 is quite old.  Not familiar with Arch Linux, but can you try one of the alpha builds, or build from our nightly codebase?
https://github.com/sqlitebrowser/sqlitebrowser/releases

**Comment by MKleusberg** [comment]
@chrisjlocke is right about trying a newer version. You'll have to compile DB4S by yourself though on Linux - but it's fairly straightforward and we can assist you if there are any problems :smile: 

In the meantime, it's also worth asking if this happens for any table or just for one or a few tables. What happens if you create a new table, save it, then edit it again and modify the data type? If it doesn't happen for all tables, can you paste the table schema in here?

**Comment by cvoinf** [comment]
I have the same problem on manjaro (arch-derivate), sqlitebrowser 3.10.1.
I've attached the database [laender9.db.zip](https://github.com/sqlitebrowser/sqlitebrowser/files/2592869/laender9.db.zip) whose table area I previously imported from a cvs-file. I wanted to change the datatype of the area.field2 from TEXT to REAL, but it crashes immediately. 
Then I tried to compile the version sqlitebrowser-3.11.0-alpha1, but it crashes too when compiling, see second file [compile-error.txt](https://github.com/sqlitebrowser/sqlitebrowser/files/2592870/compile-error.txt).

Then I used wine to run the portable version, and everything works as expected.
